srcpkgs/staticx/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'staticx'
+pkgname=staticx
+version=0.9.1
+revision=1
+archs="x86_64-musl x86_64"
```

Comment:
`archs="x86_64*"`

and why is it restricted to just this arch?

Reply:
StaticX currently works only with Linux 64-bit dynamic executables